docs/howtos/integrations/_langgraph_agent_evaluation.md

```diff
@@ -0,0 +1,550 @@
+# Building and Evaluating a ReAct Agent for Fetching Metal Prices
+
+AI agents are becoming increasingly valuable in domains like finance, e-commerce, and customer support. These agents can autonomously interact with APIs, retrieve real-time data, and perform tasks that align with user goals. Evaluating these agents is crucial to ensure they are effective, accurate, and responsive to different inputs.
+
+In this tutorial, we'll:
+
+1. Build a [ReAct agent](https://arxiv.org/abs/2210.03629) to fetch metal prices.
+2. Set up an evaluation pipeline to track key performance metrics.
+3. Run and assess the agent's effectiveness with different queries.
+
+Click the [link](https://colab.research.google.com/drive/1IsmUfCVD_pVQ7HGsKj97BXOZBGXTIF8m?usp=sharing) to open the notebook in Google Colab.
```

Comment:
use this format for linking "https://colab.research.google.com/github/explodinggradients/ragas/blob/main/<path_to_notebook>.ipynb"

Reply:
```
Fetch for https://api.github.com/repos/explodinggradients/ragas/contents/howtos/integrations?per_page=100&ref=docs failed: {
  "message": "No commit found for the ref docs",
  "documentation_url": "https://docs.github.com/v3/repos/contents/",
  "status": "404"
}

CustomError: Fetch for https://api.github.com/repos/explodinggradients/ragas/contents/howtos/integrations?per_page=100&ref=docs failed: {
  "message": "No commit found for the ref docs",
  "documentation_url": "https://docs.github.com/v3/repos/contents/",
  "status": "404"
}

    at new DO (https://ssl.gstatic.com/colaboratory-static/common/653a3dce72ffd5a401f2844866adbce8/external_binary_l10n__en_gb.js:2437:68)
    at va.program_ (https://ssl.gstatic.com/colaboratory-static/common/653a3dce72ffd5a401f2844866adbce8/external_binary_l10n__en_gb.js:2420:436)
    at ya (https://ssl.gstatic.com/colaboratory-static/common/653a3dce72ffd5a401f2844866adbce8/external_binary_l10n__en_gb.js:16:57)
    at va.next_ (https://ssl.gstatic.com/colaboratory-static/common/653a3dce72ffd5a401f2844866adbce8/external_binary_l10n__en_gb.js:14:195)
    at kaa.next (https://ssl.gstatic.com/colaboratory-static/common/653a3dce72ffd5a401f2844866adbce8/external_binary_l10n__en_gb.js:17:48)
    at b (https://ssl.gstatic.com/colaboratory-static/common/653a3dce72ffd5a401f2844866adbce8/external_binary_l10n__en_gb.js:17:309)

```

I’ve tried several different combinations of the path, but I’m still having trouble getting the notebook to connect properly. Could you provide more details or clarify the correct format for the path?